README.md

```diff
@@ -25,77 +35,84 @@ Add this dependency to your project's POM:
 
 ## Usage for DSPs
 
-See com.uid2.client.test.IntegrationExamples for example usage for DSPs.
+For example usage for DSPs, see com.uid2.client.test.IntegrationExamples.**(gwh/JN_01 where is this? Can we link?)**
```

Comment:
Yes it's https://github.com/IABTechLab/uid2-client-java/blob/master/src/test/java/com/uid2/client/test/IntegrationExamples.java

Reply:
Added, thx.